Adding some notes about composer in upgrade docs (#2370)
idno/known: README.md

### Installing

> Known is under active development and requires PHP 7+ with selected extensions, together with a supported database backend. Detailed installation instructions: <http://docs.withknown.com/en/latest/install/index.html>

Reviewer: ```suggestion
Known is under active development and requires PHP 7+ with selected extensions, together with a supported database backend. You can find detailed installation instructions here: <http://docs.withknown.com/en/latest/install/index.html>
```

#### Installing from packages

_Unofficial_ install packages, which are periodically built from the latest code, are available: <https://www.marcus-povey.co.uk/known/>

> #### Installing from Github

Reviewer: ```suggestion
#### Installing from GitHub
```

> You can opt to check out the bleeding edge code from the git repository: <https://github.com/idno/Known>

Reviewer: ```suggestion
You can opt to check out the work-in-progress development code from the git repository: <https://github.com/idno/Known>
```
